### PR TITLE
C4GT 2024 Frontend

### PR DESCRIPTION
**Pull request by riya-2206**
Have added react components in order to fetch the issue details

### CODE REVIEW

File: src/components/Sidebar.js
Comment (amit-s19) [inline src/components/Sidebar.js:1]: Please avoid using inline styling wherever possible

File: src/hooks/useParseMarkdown.js
Comment (amit-s19) [inline src/hooks/useParseMarkdown.js:1]: What does the weeklyGoals and weeklyLearnings code do?

File: src/pages/docs/2024/index.js
Comment (amit-s19) [inline src/pages/docs/2024/index.js:1]: Please show error if there exists any.

File: src/pages/docs/2024/org/index.js
Comment (amit-s19) [inline src/pages/docs/2024/org/index.js:1]: Same things here too

File: src/pages/docs/2024/index.js
Comment (riya-2206) [reply src/pages/docs/2024/index.js:1]: it is there already

File: src/pages/docs/2024/org/index.js
Comment (riya-2206) [reply src/pages/docs/2024/org/index.js:1]: in this also I am showing error

File: src/hooks/useParseMarkdown.js
Comment (riya-2206) [reply src/hooks/useParseMarkdown.js:1]: it was there to handle md I will remove this code

File: src/components/Sidebar.js
Comment (riya-2206) [reply src/components/Sidebar.js:1]: have removed it for many places will try to remove for other places also

File: src/components/OrgDescription.js
[the remark this answers is not in the document]
Comment (riya-2206) [reply src/components/OrgDescription.js:1]: I am showing error if error exists 
image

**Review by amit-s19 — APPROVED** [review]
LGTM

**Pull request merged**